src/components/learner-credit-management/data/hooks/tests/useBudgetContentAssignments.test.js

```diff
@@ -279,5 +292,66 @@ describe('useBudgetContentAssignments', () => {
         ordering: orderingQueryParam,
       },
     );
+
+    // expect(sendEnterpriseTrackEvent).toHaveBeenCalledTimes(1);
```

Comment:
Should this commented out line be un-commented or removed if no longer necessary?

Reply:
Removed 👍🏽